channels/ae.m3u

```diff
@@ -31,6 +31,8 @@ https://admdn4ta.cdn.mgmlcdn.com/adsports4/smil:adsports4.stream.smil/playlist.m
 https://admdn2.cdn.mangomolo.com/adtv/smil:adtv.stream.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="AEHawasTV.ae" tvg-name="AE | Hawas TV" tvg-country="AE" tvg-language="" tvg-logo="" group-title="",AE | Hawas TV
 https://jmc-live.ercdn.net/hawastvhd/hawastvhd_720p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="",Al Aan (720p)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x74wje5
```

Comment:
I can't open any of @LaneSh4d0w links in VLC, does it work?

Reply:
Hmm... Let me see if it works.